gobcore/message_broker/messagedriven_service.py

```diff
@@ -45,10 +48,11 @@ def _on_message(connection, service, msg):
 
     :return:
     """
-    handler = service['handler']
-
+    handler: Callable = service['handler']
     try:
         Heartbeat.progress(connection, service, msg, STATUS_START)
+        logger.configure(msg, get_logger_name(handler))
+        logger.add_message_broker_handler()
```

Comment:
Why in the try-block?

We're catching this and handling it further in the message broker, but we may not have initialised the logger, so how is that going to work?

I'd move the logger initialisation above the try-block.